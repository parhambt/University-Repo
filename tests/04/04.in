reserve_table 1 3
reserve_table 2 3
reserve_table 4 3
reserve_table 12 3
exit 1
show_table_info 3