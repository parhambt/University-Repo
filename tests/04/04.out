Paria sits at table 3
Amir Hossein sits at table 3
Mehrad enters the waiting queue of table 3
Bob enters the waiting queue of table 3
Paria exits!
Table ID: 3
People at the table: Amir Hossein, Bob
Table remaining capacity: 0
Waiting queue length: 1